application/src/main/java/de/wuespace/telestion/project/daedalus2/MessageTransformer.java

```diff
@@ -0,0 +1,116 @@
+package de.wuespace.telestion.project.daedalus2;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import de.wuespace.telestion.api.config.Config;
+import de.wuespace.telestion.api.message.JsonMessage;
+import de.wuespace.telestion.project.daedalus2.mavlink.Drehtest;
+import de.wuespace.telestion.project.daedalus2.messages.*;
+import io.vertx.core.AbstractVerticle;
+import io.vertx.core.Promise;
+
+/**
+ * Listens for {@link Drehtest} messages and converts them to further "beautified" {@link SystemT} messages.<br />
+ * <p>
+ * These new messages, among other enhancements, add hierarchy to the data (making it easier to query specific
+ * sets of data) and transforms int-encoded bit sequences to objects.<br />
+ * <p>
+ * It gets configured using {@link Configuration} with {@link Configuration#inAddress} (which it listens to) and
+ * {@link Configuration#outAddress}, which it publishes to.
+ *
+ * @author Pablo Klaschka
+ * @version 2021-05-05
+ */
+public class MessageTransformer extends AbstractVerticle {
```

Comment:
I think you might need to change some things regarding the new implementation of the MAVLink XMLs from the @Software-Team. There might be a "new" Drehtest-Message in the future.